l1-contracts/contracts/dev-contracts/test/AddressAliasHelperTest.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import "../../vendor/AddressAliasHelper.sol";
+
+contract AddressAliasHelperTest {
```

Comment:
Without wrapper test coverage doesn't work?

Reply:
No, the coverage ignores the library if it is used directly